controls/os_spec.rb

```diff
@@ -86,28 +113,38 @@
     it { should be_owned_by 'root' }
     its('group') { should eq 'root' }
     it { should_not be_executable }
-    it { should_not be_writable }
     it { should be_readable.by('owner') }
     it { should be_readable.by('group') }
     it { should be_readable.by('other') }
   end
+  describe file('/etc/login.defs'), :if => os.family == 'redhat' do
```

Comment:
because unless you are not using different values, they are default and not needed/present in login.defs of ubuntu
http://manpages.ubuntu.com/manpages/trusty/man5/login.defs.5.html

> SYS_UID_MAX (number), SYS_UID_MIN (number)
           Range of user IDs used for the creation of system users by useradd
           or newusers.

           The default value for SYS_UID_MIN (resp.  SYS_UID_MAX) is 101
           (resp.  UID_MIN-1).

which results in the same range 101-999 (except 100)

that something that is valid for other config files like ssh.
when the value is default, do we want to push it also?
issues
- clutter config files
- default can change depending on version (especially true with ssh)
- more explicit